core/dictgen/src/rootcling_impl.cxx

```diff
@@ -4519,38 +4519,31 @@ int RootClingMain(int argc,
    // Check if code goes to stdout or rootcling file
    std::ofstream fileout;
    string main_dictname(gOptDictionaryFileName.getValue());
-   std::ostream *dictStreamPtr = NULL;
-   std::ostream *dictStream = nullptr;
+   std::ostream *dictStream = &std::cout;
```

Comment:
We can make that a `std::ostream&`

Reply:
Are you sure?  Isn't it sometime swapped for a file/ofstream?

Reply:
Like @pcanal said, dictstream is assigned to fileout here - https://github.com/root-project/root/pull/5345/files#diff-64a4c9a2b092d1b125517e200783ca9fR4531 so I am not sure if that is possible..

Reply:
Indeed the assignment op is deleted. We could use the ternary op:
`std::ostream& ref = (some_condition) ? fileout : std::cout;`